internal/graph/dotgraph.go

```diff
@@ -472,3 +473,8 @@ func min64(a, b int64) int64 {
 	}
 	return b
 }
+
+func escapeDoubleQuotes(s string) string {
+	re := regexp.MustCompile(`\\([\s\S])|(")`)
```

Comment:
Would it work to simply replace `\` with `\\` and `"` with `\"`?

This could be ``strings.ReplaceAll(strings.ReplaceAll(s, `\`, `\\`), `"`, `\"`)``?

Reply:
I am not sure this is the best way, since there are occasionally other valid symbols that already have escaping backslashes (e.g. \n, \t), and this messes them up.

Reply:
I'm not sure I follow. If the comment string in a profile contains `\n` then it should be rendered as literal `\n` in pprof CLI output, in rendered graphs, everywhere. There are no escape sequences that are documented as supported in the profile.proto embedded comments.

Reply:
I think I found what the issue was here. Using your replacement line was eating a new line. The reason is that the code adds \l characters _before_ feeding it to dotgraph.go (see internal/report/report.go:1210). A simple \ replacement wouldn't really work there. Other characters such as \l are supported in graphviz (See https://graphviz.org/doc/info/attrs.html under escString). So the solution is to either make the replacement more aware or make the replacement further back in the code before any postprocessing. The advantage to the regex I had was that it left other special characters intact as well as kept the previous behavior.

Reply:
I see. I would prefer to iterate over labels and do the replacement that I suggested on each element there, before joining with \l. You can avoid the Join() call then. Basically, replace

```
fmt.Fprintf(b, ` label="%s\l"`, escapeDoubleQuotes(strings.Join(labels, `\l`)))
```

with

```
fmt.Fprint(b, ` label="`)
for _, label := range labels {
  fmt.Fprint(b, strings.ReplaceAll(strings.ReplaceAll(label, `\`, `\\`), `"`, `\"`) + `\l`)
}
fmt.Fprint(b, `"`)
```

I think it's clearer. It's harder to reason about the regular expression you used.

Reply:
Gotcha. Done, but in a separate function with separate tests for it. `\n` is also replaced with `\l` per our discussion.